<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update</class>
 <widget class="QWidget" name="Update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FrameGUI - Updates</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>7</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;It's Time to Update!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ToDoLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>266</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose an action:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateBttn">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>263</y>
     <width>93</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update Now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemindBttn">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>263</y>
     <width>116</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remind Me Later</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SkipBttn">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>263</y>
     <width>112</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip This Update</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>63</y>
     <width>536</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DetailsLabel">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>38</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Below are the details for the next new update for FrameGUI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
